src/components/inputs/InputWrapper.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <div class="row items-center">
+  <div class="row items-baseline">
     <template v-if="!attribute.definition">
       <q-input
         v-model="name"
         :label="$t('plugin.component.attribute.name')"
         class="col-4 q-px-md"
         @update:model-value="(event) => emit('update:attribute-name', event)"
       />
-      <span>:</span>
+      <span class="equals">=</span>
```

Comment:
you can use `text-subtitle1` from quasar which is visually equivalent to your custom class

Reply:
done